nuxeo-services/nuxeo-platform-convert/src/main/resources/OSGI-INF/commandline-soffice-contrib.xml

```diff
@@ -11,8 +11,9 @@
       <parameterString>--headless --norestore --writer --convert-to #{format} #{sourceFilePath} --outdir #{outDirPath}
       </parameterString>
       <testParameterString>--version</testParameterString>
-      <installationDirective>You need to install OpenOffice/LibreOffice and add soffice to the PATH environment
-        variable.
+      <winTestParameterString>--headless --norestore --writer --convert-to #{format} #{sourceFilePath} --outdir #{outDirPath}
```

Comment:
Use a command-line without `#{..}` stuff at least, and I'm not sure attempting a conversion on non-existing files is the best thing to do.